src/types/entities.ts

```diff
@@ -285,6 +285,7 @@ export const CurrentUserSchema = z.object({
     avatarSmall: z.string().nullish(),
     businessAccountId: z.string().nullable(),
     isPremium: z.boolean(),
+    premiumStatus: z.string(),
```

Comment:
If we already know what all the values will be, could we not make this an enum type?

Reply:
I was about to say that we don't use enums in this schema anywhere, but that's not true because we do that for workspace role, workspace plan, workspace plan status, upload status, etc.

The only downside is that if we ever want to add some enum values for this field we'd have to open a PR against this repo to add the new field, however 1. I don't think that'll be very often, 2. it's not that much work 😅

Hence, I'll update this PR now to switch to the enum type 👍